File: src/main/java/com/truyum/model/MenuItem.java
package com.truyum.model;



import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter @Setter @NoArgsConstructor @ToString @AllArgsConstructor
@Entity
@Table(name="menuitem")
public class MenuItem 
  {
		@Id
		long id;
		@Column(name="fname")
		String name;
		float price;
		String active;
		@Column(name="dateoflaunch")
		String dateOfLaunch;
		String category;
		@Column(name="freedelivery")
		String freeDelivery;
  }

File: src/app/model/MenuItem.ts
export class MenuItem
  {
    id:number=0;
    name:string='xxxxx';
    price:number=0;
    active:boolean=false;
    dateOfLaunch:string='';
    category:string='';
    freeDelivery:string='no';
    constructor()//id:number,name:string,price:number,active:boolean,dateOfLaunch:string ,category:string,freeDelivery:string)
     {
      this.id=0;//id;
      this.name='';//name;
      this.price=0;//price;
      this.active=false;//active;
      this.dateOfLaunch='';//dateOfLauch;
      this.category='';//category;
      this.freeDelivery='no';//freeDelivery;
     }
    public getId()
      {
        return this.id;
      }
    public getName()
      {
        return this.name;
      }
    
 }
